libs/blocks/marquee/marquee.js

```diff
@@ -69,6 +69,21 @@ function decorateText(el, size) {
   size === 'large' ? decorate(heading, 'xxl', 'xl', 'l') : decorate(heading, 'xl', 'm', 'm');
 }
 
+function decorateMultipleIconArea(iconArea) {
+  iconArea.querySelectorAll(':scope > picture').forEach((picture) => {
+    const src = picture.querySelector('img')?.getAttribute('src');
+    const a = picture.nextElementSibling;
+    if (src?.endsWith('.svg') || a?.tagName !== 'A') return;
+    
+    if (!a.querySelector('img') && !a.textContent.endsWith('.svg') && !a.textContent.includes('.svg |')) {
```

Comment:
What does `.svg |` do?

Reply:
alt text: https://github.com/adobecom/milo/pull/401

Reply:
This shouldn't be needed. We remove all that decoration before marquee gets to it.

By the time marquee.js loads, there should be no a tags with svg in them. If there are, something is wrong.

Reply:
> This shouldn't be needed. We remove all that decoration before marquee gets to it.
> 
> By the time marquee.js loads, there should be no a tags with svg in them. If there are, something is wrong.

Ah ok no problem I'll remove this check!

Reply:
Done